elasticsearch7/search-index/src/test/java/org/vertexium/elasticsearch7/Elasticsearch7SearchIndexTest.java

```diff
@@ -261,6 +267,39 @@ public void testQueryExecutionCountWhenScrollingApi() {
         assertEquals(startingNumQueries + 8, getNumQueries());
     }
 
+    @Test
```

Comment:
duplicate of the `GraphTestBase` code below. `GraphTestBase` is the base class of all graph tests so it will run for ES7